manifest.json

```diff
@@ -2,7 +2,7 @@
   "vendor": "vtex",
   "name": "minicart",
   "title": "Mini Cart",
-  "version": "2.59.0",
+  "version": "2.60.0",
```

Comment:
You don't need to change this manually, once the PR is merged, a bot will handle the release process :)